VivistaServer/Templates/outliers.liquid

```diff
@@ -0,0 +1,88 @@
+﻿<div class="pure-g center center-text">
+	<div class="pure-u-1" >
+		<h1>Outliers</h1>
+
+		<table class="pure-table table-center" id="outlierTabel">
+			<thead>
+				<tr>
+					<th>Request time in ms</th>
+					<th>Timestamp</th>
+					<th>Endpoint</th>
+					<th>Form</th>
+					<th>Query</th>
+				</tr>
+			</thead>
+			<tbody id="outlierTabelBody">
+			</tbody>
+		</table>
+	</div>
+</div>
+
+
+
+<script>
+	let outliers = JSON.parse(sessionStorage.getItem('outliers'));
+	console.log(outliers);
+	PrintOutliers();
+
+	function PrintOutliers() {
```

Comment:
Mag je hernoemen naar InsertOutliers. PrintOutliers impliceert voor mij dat die print naar een console of stdout.

Reply:
Zal ik doen